articles/iot-dps/concepts-symmetric-key-attestation.md

```diff
@@ -76,7 +76,7 @@ Replace the value of `--key` argument with the **Primary Key** from your enrollm
 Replace the value of `--registration-id` argument with your registration ID.
 
 ```azurecli
```

Comment:
Please also update the link to the CLI reference in line 72. 

It should point to /cli/azure/iot/dps/enrollment-group#az-iot-dps-enrollment-group-compute-device-key

Reply:
Thank you, Kelly, for pointing this out. I have updated the link.